gordo_components/client/client.py

```diff
@@ -618,6 +622,15 @@ def predict(
             Dataframe of required tags and index reflecting the datetime point
         """
 
+        # We want to adjust for any model offset. If the model outputs less than it got in, it requires
+        # extra data than what we're being asked to get predictions for. Double the expected model offset
```

Comment:
Why double compared to e.g. "+ 5"? Why should no offset mean 0 buffer, but 144 offset should mean 288 buffer?

Reply:
Would `+ 5` be more acceptable? I did double of the offset for no particular reason, but can see with an offset of 0, it isn't an optimal choice indeed.

Reply:
I think some kind of constant offset is more reasonable yeah. But when we change the dataset so it goes back and gets data so it can deliver data from the actual start-date then I guess we dont need this anymore?